### DISCUSSION TITLE
海外回国，接入国内设备passwall2的服务器端

### DISCUSSION THREAD

**Discussion by Travel2Here** [Openwrt-Passwall/openwrt-passwall2#567] [General]
如何配置为佳？
目前选的xray socks 出站节点关闭，不确定这线路是会分流还是就直连了？或是走的默认节点？

目的，人在海外，希望接入这个服务器节点是完全直连 不进行任何分流 请教点拨~

**Comment by handsomeji** [comment]
![image](https://github.com/user-attachments/assets/6d26ddad-deb4-44ef-b808-378e429a2ef9)

**Reply by nitobit** [reply]
3X-UI: REVERSE PROXY - Tunnel to the ROUTER (X-ray, VLESS) https://xtls.github.io/ru/config/reverse.html#reverseobject it is possible to set up a reverse proxy to use the admin panel's access to the router via the x-ui server
